composer.json

```diff
@@ -43,7 +47,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.4.x-dev"
+            "dev-master": "2.0.x-dev"
```

Comment:
Updating the alias. Do we still want to create a 1.0 branch? I say let’s just stick to our old plan of not maintaining parallel branches (as long we can get away with doing so) and upping master to 2.0.